intl/en/sl-deploy.txt

```diff
@@ -0,0 +1,64 @@
+usage: {{%MAIN% [options] [URL [PACK|BRANCH]]}}
+
+Deploy a node application to a StrongLoop process manager
+
+Options:
+  {{-h,--help}}          Print this message and exit.
+  {{-v,--version}}       Print version and exit.
+  {{-s,--service SVC}}   Deploy to service SVC.
```

Comment:
Who is our tech editor? Is it Rand?

We are responsible for the command line usage and making it correct, particularly from a coding/markup point of view. Rand sometimes has suggestions for improving our usage messages, but he's not going to proofread all this stuff, and not going to translate it to see if things get translated that should not have been, I think that kind of quality is up to us.